Packs/DomainToolsIrisDetect/Integrations/DomainToolsIrisDetect/DomainToolsIrisDetect.yml

```diff
@@ -0,0 +1,1353 @@
+  options:
+  - Import Indicators Only
+  - Create Incidents and Import Indicators
+  required: false
+  type: 15
+- additionalinfo: Monitors the Iris Detect endpoint for additions to domains added to the blocklist. Additions can be made via this app or the Iris Detect UI. This is useful for teams wishing to route triage domains to firewall software for blocking. If selected, each pull will create a new incident every time the enrichment is run, with the new domains attached as indicators to the incident. Whois and DNS information is preserved in comments.
+  defaultvalue: Create Incidents and Import Indicators
+  display: Enabled on Blocked Domains
+  hidden: false
+  name: blocked_domains
+  options:
+  - Import Indicators Only
+  - Create Incidents and Import Indicators
+  required: false
+  type: 15
+- additionalinfo: Optionally specify a risk score range to triage higher risk indicators to different routing. A higher number indicates higher confidence a domain is likely to be used for malicious purposes.
+  display: Risk score Ranges
+  hidden: false
+  name: risk_score_ranges
+  options:
+  - 0-0
+  - 1-39
+  - 40-69
+  - 70-99
+  - 100-100
+  required: false
+  type: 15
+- additionalinfo: Includes DNS and whois data in the response.
+  defaultvalue: 'true'
+  display: Include Domain Data
+  hidden: false
+  name: include_domain_data
+  required: false
+  type: 8
+- additionalinfo: For the first time the enrichment is run, specify how far back should it pull indicators. First Fetch timestamp, Default is 3 days. The maximum time range is 30 days.
+  defaultvalue: 3 days
+  display: First fetch timestamp (example 7 days)
+  name: first_fetch
+  required: false
+  type: 0
+- additionalinfo: This is a required field by XSOAR and should be set to 3, one for each possible feed type new, changed, blocked.
+  defaultvalue: '3'
+  display: Maximum number of incidents to fetch
+  hidden: true
+  name: max_fetch
+  required: false
+  type: 0
+- additionalinfo: Trust any certificate (not secure)
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- additionalinfo: Use system proxy settings
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: '240'
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  required: false
+  type: 19
+  section: Collect
+  advanced: true
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- defaultvalue: 'true'
+  display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+description: DomainTools is an essential component in the security stack of mature enterprises and performance-driven security teams.
+display: DomainTools Iris Detect
+name: DomainToolsIrisDetect
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: List of Iris Detect domain IDs to escalate. The domain ID can be found in the API response when querying the "new" endpoint.
```

Comment:
Please add here the name of the command that can retrieve the domain ids, and update the other command's description as well, and the matching README description.